common/http/src/test/java/io/helidon/common/http/SetCookieTest.java

```diff
@@ -0,0 +1,59 @@
+package io.helidon.common.http;
+
+import org.junit.Test;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.text.SimpleDateFormat;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class SetCookieTest {
+
+    @Test
+    public void testExpireswithInstant() {
+        SetCookie setCookie = new SetCookie("foo", "bar");
+        SimpleDateFormat sdf = new SimpleDateFormat("EEE, d MMM yyyy HH:mm:ss Z");
+        assertEquals("foo=bar; Expires=" + sdf.format(new Date()), setCookie.expires(Instant.now()).toString());
```

Comment:
🚫 `new Date()` and `Instant.now()` might not return the same number. Take them from the same instance.
Same below.

Reply:
new Date was changed to Instant.now()